mods/default/chests.lua

```diff
@@ -22,7 +22,7 @@ function default.chest.get_chest_formspec(pos, size)
 			x_player, size.y + 0.85,
 			x_player, size.y + 2.08,
 			spos) ..
-		default.get_hotbar_bg(0, 0.85 + size.y)
+		default.get_hotbar_bg((fs_size.x - 8) / 2, 0.85 + size.y)
```

Comment:
Should be `x_player` for consistency since it's related to the player inventory size. Surely this will work just fine, but confuses a bit when reading the code.

Reply:
OK I'll change it.